lib/matplotlib/figure.py

```diff
@@ -1001,6 +1002,142 @@ def add_subplot(self, *args, **kwargs):
         self.stale = True
         return a
 
+    def add_subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
```

Comment:
It's a bit like `Figure.suptitle`: I don't like it because I don't see why it should be `fig.add_subplot` but `fig.subplots`, just like I don't see why it should be `ax.set_title` but `fig.suptitle`.